### ISSUE TITLE
cellpose gui or using python code producing very strange results compared to cellpose.org

### ISSUE THREAD

**Issue by htahir2** [MouseLand/cellpose#442]
I am new to cellpose and tried it on the cellpose.org with a sample image and it worked amazing. Since I need to work with many images, I installed cellpose through conda environment (tried both on mac and windows) and used the same image to test if I get the same results. I tried both GUI as well as python code but results are very different from the web based segmentation. 
I do not understand where is the issue. I tried both models cyto and cyto2 but results are different. When I try on the website with a cell diameter 30, I get very nice results. I also tried the same code on google colab and there results are different from local installation as well as from the web based output.
Are there any differences in the pretrained models? Can someone guide me to find the issue?

I tried the following options:
Using command line:
` python -m cellpose --dir ~/Downloads/test/ --pretrained_model cyto --chan 0  --save_png --verbose`

On google colab or local installations:
```
import numpy as np
import matplotlib.pyplot as plt
from skimage import io
from cellpose import models, core, plot

# use_GPU = core.use_gpu()
# print('>>> GPU activated? %d'%use_GPU)

img = io.imread('/Users/ht/Downloads/test/prog1_slice1_cropped.png')
plt.imshow(img, cmap='gray')

# model = models.Cellpose(gpu=use_GPU, model_type='cyto') ## for google colab
model = models.Cellpose(gpu=False, model_type='cyto') ## for local installation

masks, flows, styles, diams = model.eval(img, diameter=30, channels=[0,0])
plt.imshow(masks, cmap='gray')

fig = plt.figure(figsize=(12,5))
plot.show_segmentation(fig, img, masks, flows[0], channels=[0,0])
plt.tight_layout()
plt.show() 
```
here is the image that I tried
![prog1_slice1_cropped](https://user-images.githubusercontent.com/77272915/154061588-734d783d-becf-4d90-8ce5-4c6ae412ec1f.png)
:

**Comment by kevinjohncutler** [comment]
Could you post pictures of that the output masks look like in both cases (cellpose.org and local)?

**Comment by htahir2** [comment]
Hello,

I think I found the issue. On the website, images were resized to 512 x 512 max, but locally i was not doing it. So I need to increase the cell diameter for the images without resizing. I guess you can close the issue or if you want, you can delete it.

Regards,
Hannan
________________________________
